examples/alternate-receive/alternate_receive.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/lang.runtime;
+
+public function main() {
+    // Both arguments passed to the function are valid URLs.
+    // Thus alternate receive in worker `w3` sets the
+    // first value it rececives from a worker as the result.
+    fetchFirst("https://postman-echo.com/get?worker=w1", "https://postman-echo.com/get?worker=w2");
```

Comment:
Shall we change the signature to retrieve the value and print here instead

```suggestion
    string? firstFetched = getFirstFetched("https://postman-echo.com/get?worker=w1", "https://postman-echo.com/get?worker=w2");
```

Reply:
In that case should we do inline printing of the value, than assigning the value to a variable?
Wouldn't having the `println` in the common function be enough, because we are only having the println function once in the code.

Reply:
No, we can break into two. println in the next line.